src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1112,5 +1112,17 @@ public override void VisitRangeCaseClause(IRangeCaseClauseOperation operation)
                 AssertEx.Equal(new[] { operation.MinimumValue, operation.MaximumValue }, operation.Children);
             }
         }
+
+        public override void VisitFlowCapture(IFlowCaptureOperation operation)
+        {
+            Assert.Equal(OperationKind.FlowCapture, operation.Kind);
+            Assert.Same(operation.Value, operation.Children.Single());
+        }
+
+        public override void VisitFlowCaptureReference(IFlowCaptureReferenceOperation operation)
```

Comment:
`Assert.True(operation.IsImplicit)`?

Reply:
We may also want to verify that each flow capture ID has exactly one corresponding `IFlowCaptureOperation` and `IFlowCaptureReferenceOperation`.

Reply:
> Assert.True(operation.IsImplicit)?

The value is hardcoded, but I guess having an assert doesn't hurt, especially if something going to change in the future. Will add in the next PR.

---
In reply to: [163293753](https://github.com/dotnet/roslyn/pull/24358#discussion_r163293753) [](ancestors = 163293753)

Reply:
> We may also want to verify that each flow capture ID has exactly one corresponding  IFlowCaptureReferenceOperation.

This is a wrong expectation. A captured result can be referenced multiple times.

---
In reply to: [163294236](https://github.com/dotnet/roslyn/pull/24358#discussion_r163294236) [](ancestors = 163294236)